plugins/file_logs.yaml

```
@@ -0,0 +1,78 @@
+version: 0.0.1
+title: File
+description: Log parser for generic files
+parameters:
+  - name: file_path
+    type: "[]string"
+    default: []
```

I think this should be a required parameter with no default.
```suggestion
  - name: file_path
    type: "[]string"
    required: true
```